crates/primitives/src/block.rs

```diff
@@ -427,7 +427,7 @@ impl SealedBlock {
             return Err(GotExpected {
                 got: calculated_root,
                 expected: self.header.transactions_root,
-            })
+            });
```

Comment:
```suggestion
            })
```
pls revert unrelated formatting change (use `cargo +nightly fmt` instead of `cargo fmt`)

Reply:
Yeah, I have update the fmt setting of rust-analyzer :)